piety-promoted-v3/modernized/11-chapter-11.adoc

```diff
@@ -936,13 +937,14 @@ he found himself ill; and leaning, through weakness,
 on the rail of the ministers`' gallery,
 he repeated pathetically those well-known lines of Addison,
 
-"`My life, if you preserve my life,
+[verse]
```

Comment:
I wasn't sure whether I should keep the quotes around these lines since it's within the "verse" thingy?

Reply:
I am totally unable to decide what I think is better, haha.  So I'll just go with what you did here :)